test/cluster/test_hints.py

```
@@ -25,14 +27,95 @@
+
 async def get_hint_metrics(client: ScyllaMetricsClient, server_ip: IPAddress, metric_name: str):
+    return await get_metric(client, server_ip, f"scylla_hints_manager_{metric_name}")
+
+
+async def get_all_hint_metrics(client: ScyllaMetricsClient, server_ip: IPAddress) -> dict:
     metrics = await client.query(server_ip)
-    return metrics.get(f"scylla_hints_manager_{metric_name}")
 
-async def create_sync_point(client: TCPRESTClient, server_ip: IPAddress) -> str:
-    response = await client.post_json("/hinted_handoff/sync_point", host=server_ip, port=10_000)
+    def resolve_metric(name: str):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `resolve_metric`

(ANN202)